File: Java/Simple Operations And Calculations/Extra/TrainingLab.java
import java.util.Scanner;

public class TrainingLab {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double h = Double.parseDouble(scanner.nextLine());
        double w = Double.parseDouble(scanner.nextLine());

        int rows = (int)(Math.floor((h * 100)/120));
        int seatsPerRow = (int)(Math.floor((w * 100 - 100)/70));

        int seats = rows * seatsPerRow - 3;
        System.out.println(seats);
    }
}

File: C#/02. Simple Operations and Calculations/Extra/Training Lab.cs
using System;

namespace Training_Lab
{
    class Program
    {
        static void Main(string[] args)
        {
            double w = double.Parse(Console.ReadLine()) * 100;
            double h = double.Parse(Console.ReadLine()) * 100;

            h -= 100;
            double rows = Math.Floor(h / 70);
            double columns = Math.Floor(w / 120);
            double workingPlaces = rows * columns - 3;

            Console.WriteLine(workingPlaces);
        }
    }
}
